src/main/java/it/infuse/jenkins/usemango/model/Response.java

```diff
@@ -0,0 +1,58 @@
+package it.infuse.jenkins.usemango.model;
+
+import com.google.api.client.json.GenericJson;
+import com.google.api.client.util.Key;
+
+import java.util.List;
+
+public class Response<T> extends GenericJson {
```

Comment:
Might be better to call it `PagedResponse` or something along those lines to clarify it's not a simple response.